WMass/python/plotter/plotUtils/utility.py

```diff
@@ -367,16 +377,16 @@ def getEntries(inp, mirror=False):
         nentries *= 2
     allentries = range(nentries)
     return allentries
-
-def mirrorGroups(inp, addMirror=True, maxnum=-1):
+1
+def mirrorGroups(inp, addMirror=True, maxnum=-1, indexOffset=0):
     allentries = getEntries(inp, addMirror)
     if maxnum > 0:
         allentries = allentries[:maxnum*2]
     mid = int(len(allentries)/2)
-    return {str(i) if len(allentries) != 2 else "" : v 
+    return {str(i+indexOffset) if len(allentries) != 2 else "" : v 
```

Comment:
I don't think you want the index offset in the name, do you? This would make the first index labeled as pdf2 if the index offset is 2

Reply:
actually passing indexOffset = 1 I get the nuisances being named pdf1, pdf2, and so on (this is because the histogram with variations contains the nominal, which falls in the first bin (indexed as 1 in root), but in order not to have a pdf0 being just the nominal I had to unpack from histogram bin 2 onwards (using the argument of util.makeVariationHistsForCharge) and thus when they are associated to the label I need the label to start from 1 to have the parameter be named pdf1, ...